owncloudApp/src/main/res/xml/settings_picture_uploads.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ownCloud Android client application
+
+  Copyright (C) 2021 ownCloud GmbH.
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License version 2,
+  as published by the Free Software Foundation.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<PreferenceScreen xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <SwitchPreferenceCompat
+        app:iconSpaceReserved="false"
+        app:key="enable_picture_uploads"
+        app:summary="@string/prefs_camera_picture_upload_summary"
+        app:title="@string/prefs_camera_picture_upload" />
+    <Preference
+        app:iconSpaceReserved="false"
+        app:key="picture_uploads_path"
+        app:title="@string/prefs_camera_picture_upload_path_title" />
+    <CheckBoxPreference
+        app:iconSpaceReserved="false"
+        app:key="picture_uploads_on_wifi"
+        app:title="@string/prefs_camera_picture_upload_on_wifi" />
+    <Preference
+        app:iconSpaceReserved="false"
+        app:key="picture_uploads_source_path"
+        app:title="@string/prefs_camera_upload_source_path_title" />
+    <ListPreference
+        app:iconSpaceReserved="false"
+        app:defaultValue="NOTHING"
+        app:dialogTitle="@string/prefs_camera_upload_behaviour_dialogTitle"
```

Comment:
We will use camelCase when writing in kotlin, but in XML files, we will use underscores.

```suggestion
        app:dialogTitle="@string/prefs_camera_upload_behaviour_dialog_title"
```